### PR TITLE
feat: footer component

### PR DESCRIPTION
**Pull request by ndom91**
- Added footer component mirroring our existing one
- Used same `max-width` and what not as navbar, so they match in width always

Misc:
- `.gitignore` the `tsconfig.tsbuildinfo` file
- `tsconfig.json` `jsx: "react-jsx"` to get rid of error regarding UMD
- Added `tailwind.config.js` `darkMode: 'class'`, since `dark:*` classes weren't really working yet 
- 

image

### CODE REVIEW

File: pages/overview/_meta.js [hunk 1]
```diff
@@ -16,7 +21,10 @@ export default {
   },
   deployment: "Deployment",
   typescript: "Typescript",
-  "migrating-to-v5": "Migrate to NextAuth@5",
+  "-- More": {
+    type: "separator",
+    title: "More",
+  },
```
Comment (0ubbe) [inline pages/overview/_meta.js:27]: In theory, we were going to move these two pages, **Contributors** and **Sponsors**, to the footer, so the sidebar for "Getting started" will stay quite lean, hence maybe no need for the sections 💭

**Review by 0ubbe — APPROVED** [review]
Noice 🙌🏽  💯 , small suggestion around the sidebar items on "Getting started" ☕

File: pages/overview/_meta.js (hunk 1 above)
Comment (ndom91) [reply pages/overview/_meta.js:27]: Yeah i saw that note, added them to the footer too I'm that PR of mine, but it doesn't matter to me. If y'all wanna remove them here, just go for it 👍

Comment (ndom91) [reply pages/overview/_meta.js:27]: Ahh haha this is that PR 😅 too many tabs open haha. But yeah feel free to remove them there if y'all want. Balazs had a little commet on slack that maybe we leave them here then too 🤔

**Comment by 0ubbe** [comment]
Screenshot 2024-01-17 at 21 37 13

If it's easy enough, would be cool to align these, but my hard because of the Nextra layout, I can try otherwise in a separate PR 📐

**Comment by ndom91** [comment]
> If it's easy enough, would be cool to align these, but my hard because of the Nextra layout, I can try otherwise in a separate PR 📐

What do you mean exactly? Its algined with the header using the same width/padding classes (`max-w-[90rem]` and `px-6`) and seems lined up with the outside of the text there too :thinking: 

image

**Pull request merged**